### PR TITLE
KEYCLOAK-5230 Add indices to FED_* tables

### PR DESCRIPTION
**Pull request by hmlnarik**

### CODE REVIEW

**Comment by hmlnarik** [comment]
@vramik Could you please check this PR on supported DBs?

File: model/jpa/src/main/resources/META-INF/jpa-changelog-3.4.0.xml [hunk 1]
```diff
@@ -47,4 +47,63 @@
...
+            <column name="REALM_ID" type="VARCHAR(36)" />
+            <column name="USER_ID" type="VARCHAR(255)" />
+        </createIndex>
+
+        <createIndex tableName="FED_USER_GROUP_MEMBERSHIP" indexName="IDX_FED_USER_GROUP_MEMBERSHIP">
+            <column name="USER_ID" type="VARCHAR(255)" />
+            <column name="GROUP_ID" type="VARCHAR(36)" />
+        </createIndex>
+
+        <createIndex tableName="FED_USER_GROUP_MEMBERSHIP" indexName="IDX_FED_USER_GROUP_MEMBERSHIP_RU">
```
Comment (vramik) [inline model/jpa/src/main/resources/META-INF/jpa-changelog-3.4.0.xml]: indexName IDX_FED_USER_GROUP_MEMBERSHIP_RU is too long on Oracle DB, the limit is 30

File: model/jpa/src/main/resources/META-INF/jpa-changelog-3.4.0.xml [hunk 2]
```diff
@@ -47,4 +47,63 @@
...
+            <column name="REALM_ID" type="VARCHAR(36)" />
+            <column name="USER_ID" type="VARCHAR(255)" />
+        </createIndex>
+
+        <createIndex tableName="FED_USER_REQUIRED_ACTION" indexName="IDX_FED_USER_REQUIRED_ACTION">
+            <column name="USER_ID" type="VARCHAR(255)" />
+            <column name="REQUIRED_ACTION" type="VARCHAR(255)" />
+        </createIndex>
+
+        <createIndex tableName="FED_USER_REQUIRED_ACTION" indexName="IDX_FED_USER_REQUIRED_ACTION_RU">
```
Comment (vramik) [inline model/jpa/src/main/resources/META-INF/jpa-changelog-3.4.0.xml]: indexName IDX_FED_USER_REQUIRED_ACTION_RU is too long on Oracle DB, the limit is 30

**Review by vramik — CHANGES_REQUESTED** [review]
There are too long identifiers for Oracle DB

**Comment by hmlnarik** [comment]
@vramik Thanks, this should be fixed in the new commit, could you please recheck?

**Comment by vramik** [comment]
@hmlnarik tested and passed, thank you.

**Comment by hmlnarik** [comment]
@vramik Thank for checking

**Pull request merged**